src/main/java/org/fundacionjala/coding/escarleth/movies/Movie.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.escarleth.movies;
+
+/**
+ * Powered by Escarleth L.
+ */
+public abstract class Movie {
+    private final String title;
+
+    /**
+     * Constructor Movie Class.
+     *
+     * @param title of the Movie.
+     */
+    public Movie(String title) {
+        this.title = title;
+    }
+
+    /**
+     * @return title of the Movie.
+     */
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     * @param daysRented is the quantity of rental days.
+     * @return the total amount .
+     */
+    abstract double determineRentalAmount(int daysRented);
+
+    /**
+     * @param daysRented is the FrequentRenterPoints.
+     * @return FrequentRenterPoints .
+     */
+    int determinateFrequentRenterPoints(int daysRented) {
+        return 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3400)